insights/parsers/systemctl_show.py

```diff
@@ -20,13 +20,17 @@
 
 SystemctlShowPulpCelerybeat - command ``systemctl show pulp_celerybeat``
 ------------------------------------------------------------------------
+
+SystemctlShowHttpd - command ``systemctl show httpd``
+-----------------------------------------------------
 """
 
+from insights.specs import Specs
+
 from .. import LegacyItemAccess
 from .. import Parser
 from .. import parser
 from ..parsers import split_kv_pairs
-from insights.specs import Specs
```

Comment:
@TripleJQK you could just change this line to `from ..specs import Specs`.  Its not clear why you moved it above the other imports.

Reply:
modified